utils/context.py

```diff
@@ -238,6 +237,8 @@ def reset(self):
             self.irods_environment.reset()
             filepath = path.LocalPath(self.irods_env_file).expanduser()
             self.irods_environment.filepath = filepath
+        if self.irods_connector:
+            self.irods_environment.reset()
```

Comment:
This is already done.  I don't understand why this has been added.

Reply:
A recent PR resets the **irods_connector** instead of deleting it.  These two lines were also removed.  I hope that did not restore the bug.

Reply:
No actually that is very much needed to prevent yet another bug ;) So far all seems really fine.